src/main/java/orca/flukes/ndl/ManifestLoader.java

```diff
@@ -388,85 +387,17 @@ public void ndlLinkConnection(Resource l, OntModel m,
 			setRequestGuid(l, ml);
 			
 			nodes.put(getTrueName(l), ml);
-			
-			// special case handling - if storage is one side of the link
-			// (only for shared vlan storage), then we only remember
-			// interfaces with no ip addresses on them (others are duplicates; 
-			// we save the equivalence relationship to lookup IP later) /ib 09/18/14
-			boolean sharedVlanStorageLink = false;
```

Comment:
6. If we don't need sharedVlanStorage link, we don't need these declarations and the loop following it.